docs/guide/topics/permissions.rst

```diff
@@ -0,0 +1,222 @@
+.. currentmodule:: discord
+
+.. _guide_topic_permissions:
+
+
+Permissions
+============
+
+Permissions are the bread and butter of how control over a Discord server (which will from now on be referenced as a "guild" to align with Discord's official API terminology).
+
+Controlling permissions is necessary for securing and controlling how your guild functions and operates.
+
+
+Permission hierarchy explanation
+---------------------------------
+
+Discord permissions work on a hierarchy system. This means that roles placed logically above each other (in the Roles UI) have descending order of precedence.
+
+If you have a role list, like so:
+
+- ``Administrator (#1)``
+- ``Manager (#2)``
+- ``Moderator (#3)``
+- ``@everyone (#4)``
+
+This means that Administrator role (someone who has it) can kick, ban and timeout Manager.
+Manager **cannot** kick, ban or timeout Administrator.
+
+discord.py is aware of this and has implemented the ``__gt__`` and other methods on :class:`Role`, which allows you to do the following:
+
+.. code-block:: python3
+
+    if administrator_role > manager_role:
+        # The administrator role is greater in the hierarchy than manager.
+
+One small caveat to note with the discord permissions system, is that in the case of overwrites, explicit allow will always overwrite explicit deny.
```

Comment:
This isn't entirely accurate because a member overwrite deny takes precedence over a role overwrite allow. Maybe also list the whole overwrites hierarchy while we're at it and not just allow > deny

```suggestion
One small caveat to note with the discord permissions system, is that in the case of overwrites, the highest overwrite in the hierarchy will be applied. The hierarchy is as follows:

1. Member allow
2. Member deny
3. Role allow
4. Role deny
5. @everyone allow
6. @everyone deny
```

Reply:
I take no credit for this suggestion, other than moving the comma to correct the appositive clause.

```suggestion
One small caveat to note with the discord permissions system is that, in the case of overwrites, the highest overwrite in the hierarchy will be applied. The hierarchy is as follows:
1. Member allow
2. Member deny
3. Role allow
4. Role deny
5. @everyone allow
6. @everyone deny
```